packages/simple63/simple63.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Integer compression and decompression module"
+description:
+  "Simple63 implements a module for compression and decompression of
+   non-negative integer sequences. It is similar to the Simple-8b
+   technique, adapted to work with OCaml's 63-bit integers."
+maintainer: ["mika@illouz.net"]
+authors: ["Mika Illouz"]
+license: "BSD"
+homepage: "https://github.com/barko/simple63"
+doc: "https://barko.github.io/simple63/"
+bug-reports: "https://github.com/barko/simple63/issues"
+depends: [
+  "dune" {> "1.5"}
+  "ocaml" {>= "4.08.0"}
+  "ppx_inline_test"
```

Comment:
This is required even if you don't run the tests?

Reply:
No. I'll add {with-test}, change dune to {> "2.6"} and resubmit. Thx for reviewing.